### ISSUE TITLE
Traffic light system for the status of regions

### ISSUE THREAD

**Issue by osmers** [digitalfabrik/integreat-cms#3012]
### Motivation
Inspired by the Excel sheet that we keep, to determine how well the municipalities are doing, we want to incorporate the same metric into the network management of the CMS. This serves a dual purpose.
1. We have data that is always up to date bcs it draws the necessary input directly from the system
2. We avoid about 10 hours of work every quarter to update the list manually

### Proposed Solution
Take the columns and calculations from the excel sheet and design the same/a similar table in the network management that is fed by the data from each region. 

### Alternatives
Continue working with excel

### User Story
As a service team member, I want to have up to date data on how the municipalities are doing in order to determine who needs support and in order to see while on the phone with a municipality, how they are doing.

### Additional Context
We would need the following numbers from each region
From the to do board: 
- number of outdated pages
- number of broken links
- number of pages with HIX below 15

From the page tree:
- number of pages in total in the region

From the Translation report:
- number of missing/outdated translations (if possible it would be great to add the number of pages from the diagram, but alternatively the number of words in the table works too (but then we need some more information like number of words in the region)

From the region settings: 
- yes or no whether the extra package for 1 Mio words is activated

From language tree:
- number of languages besides German/root language

![image](https://github.com/user-attachments/assets/d0acefc6-286f-4c9c-8feb-a87a8240da6c)

### Design Requirements

**Comment by osmers** [comment]
Here is the list currently being used: https://nextcloud.tuerantuer.org/index.php/f/6502587
Be careful with the calculations and everything so nothing breaks :)

**Comment by david-venhoff** [comment]
Blocked on 3053

**Comment by timobrembeck** [comment]
Is this a duplicate of #78 (or are they at least partially overlapping)? Or are they just linked to each other?

**Comment by david-venhoff** [comment]
I think they just unfortunately share the same name, but are about entirely different things :sweat_smile: 
The traffic light system mentioned here is about how well we think the municipalities are doing, while the traffic light system of #78 is about storage usage

**Comment by JoeyStk** [comment]
@osmers 
The first iteration of the table is online on cms-test now :)
However it's currently only possible to navigate by adding "?page=2" or by increasing the number of pages shown per view by adding "?size=50" to the URL. 🙈 Could you please check it out and list down a few things you wish we would improve for the next iteration? :)
One thing I was wondering about - Is it okay that currently only the name of the region is shown (not specified by what type of region it is). So LK Munich and city of Munich are both listed as simply Munich. I assume you can differentiate the two by your knowledge of the context - but maybe it's annoying and/or doesn't work for all instances (same issue for Augsburg & Schwäbisch Hall)

**Comment by osmers** [comment]
@JoeyStk thanks for your detailed questions :) 

I think it would really help, if we had the information which kind of administrative region is municipality is. Also, if the name of the municipality was linked to their dashboard, that might be useful. 

The percentage calculation was just used by me to be able to assign rank to their numbers. I.e. I looked at the ratio of outdated pages/pages with low HIX value to total pages and that percentage served to determine whether they were in the green, light green, yellow, orange or red category. If you can do that in the background, that's good enough for me.
All three categorizations (HIX, outdated pages and broken links) combined together to give me the overall color of the municipality. I can take you through it if that helps :) 

It might be nice to be able to sort the list according to the new sorting feature we are implementing anyways. And as you mentioned, we'd definitely need the possibility to extend the list and also have pagination.

Apart from all that, the most important thing in my opinion would be the color coding of the municipalities...

**Comment by JoeyStk** [comment]
This needs fixing soon: 
- [ ] missing pagination
- [ ] missing administrative division in both table and CSV

Next iteration(s):
- [ ] Add row HIX to the table
- [ ] Add row outdated page to the table
- [ ] Add row broken links to the table
- [ ] Add color coding

**Comment by osmers** [comment]
@JoeyStk do you need information on the color coding?

**Comment by JoeyStk** [comment]
There has been work on this branch: `https://github.com/digitalfabrik/integreat-cms/tree/feature/region-condition-3`. I'm unsure if my work is good or not. Please check it out, if you want, to see if it's usable.

**Comment by JoeyStk** [comment]
As far as I understood @dkehne during the last conference this can be pushed way back as it's not highly relevant to the municipalities. Did I get this right?

**Comment by osmers** [comment]
Closing as it was integrated into the CRM system
